test/e2e/knative_serving_test.go

```diff
@@ -41,19 +39,20 @@ func TestKnativeServing(t *testing.T) {
 		}
 	})
 
+	// TODO: enable again when readiness check is available for the serverless operator
 	// Check the status of the serverless operator deployment
-	if err := test.CheckDeploymentScale(caCtx, test.OperatorsNamespace, "knative-openshift", 1); err != nil {
-		t.Fatalf("Failed to verify the operator deployment: %v", err)
-	}
+	//if err := test.CheckDeploymentScale(caCtx, test.OperatorsNamespace, "knative-openshift", 1); err != nil {
```

Comment:
If we go with the suggestion below, this can just be dropped (it can be dropped anyway, the CSV won't be ready if this is not ready)